avocado/utils/network/interfaces.py

```diff
@@ -267,7 +245,7 @@ def save(self, ipaddr, netmask):
 
         self._write_to_file("{}/{}".format(path, filename),
                             {'TYPE': self.if_type,
-                             'BOOTPROTO': 'none',
+                             'BOOTPROTO': 'static',
```

Comment:
So, with your force-push, you changed the approach of the initial PR. I'm not sure how replacing none by static will affect other distros. Did you tests this on other distros? Can you explain a little bit about this?

Reply:
Hi Beraldo,
I tested with BOOTPROTO=static in SuSE and RHEL distros and IP was successfully assigned when the interface comes up. But when BOOTPROTO=none , interface fails to come up in a SuSE OS. But the approach of using a dictionary may be a good one here  in case we plan to extend this for Bond interfaces or use a different distro. Changing the BOOTPROTO=none is just a immediate approach I used in my tests.

Reply:
Ok, thanks @vaishnavibhat, I'm ok with that. But still waiting for the changes on my comment about the blank lines.